### ISSUE TITLE
The dependency tzinfo-data (>= 0) will be unused by any of the platforms Bundler is installing for.

### ISSUE THREAD

**Issue by javierojeda94** [rails/rails#27830]
### Steps to reproduce

I just installed ruby via rvm and rails (followed [this](http://railsapps.github.io/installrubyonrails-mac.html) tutorial). 

When I run ```rails new my_app``` the app should install perfectly fine with just green output but I have this message:

````
The dependency tzinfo-data (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for ruby but the dependency is only for x86-mingw32, x86-mswin32, x64-mingw32, java. To add those platforms to the bundle, run `bundle lock --add-platform mingw, mswin, x64_mingw, jruby`.
````

```rails -v```
```Rails 5.0.1```

```ruby -v```
```ruby 2.4.0p0 (2016-12-24 revision 57164) [x86_64-darwin16]```

**Comment by jeremy** [comment]
See https://github.com/bundler/bundler/issues/5337

**Comment by uknable** [comment]
As a temporary fix I installed all of the gems individually:

`gem install sqlite3 puma sass-rails uglifier coffee-rails jquery-rails turbolinks jbuilder byebug web-console listen spring spring-watcher-listen`

Running `rails s` after that gives me this error:

`Could not find gem 'listen (~> 3.0.5)' in any of the gem sources listed in your Gemfile.
Run "bundle install" to install missing gems.`

This is despite my listen gem being 3.1.5.

So then I went into the Gemfile and removed the version requirement:
`gem 'listen', ~> 3.0.5` to `gem 'listen'`

I can now get to the Yay! page when i run `rails s`
I'm running Ubuntu 16.04
